### ISSUE TITLE
Is there a way to higher the sensitivity of the Analogsticks?

### ISSUE THREAD

**Issue by skylabhalo** [DanielOgorchock/joycond#94]
It's hard playing 3ds Games like Zelda with the joy cons cause most of the time link isn't running and spins just don't work most of the time so i have to do it the manually way by holding b which is slower. Other than that I love this plugin. My deadzone in citra already is on 0% so it's not that I just want to know if you could change the sensitivity

**Comment by dos1** [comment]
I don't think this daemon is concerned with things like dead zones at all, configuring that isn't part of its job.
